### PR TITLE
Some themes from Cyberpunk 2077 and Cyberpunk: Edgerunners

### PR DESCRIPTION
**Pull request by rexgraystone**
## Discord

Zbhadboi#0565

## Testing

python3 ./scripts/gen_theme_previews.py Cyberpunk/2077/yaml/
Generating for Cyberpunk_2077_-\_Night_City.yaml
Generating for Cyberpunk_2077_-_V.yaml

python3 ./scripts/gen_theme_previews.py Cyberpunk/Edgerunners/yaml/
Generating for Edgerunners_-\_David.yaml
Generating for Edgerunners_-_Lucy_and_David_Jump.yaml

### CODE REVIEW

**Comment by elviskahoro** [comment]
@rexgraystone hey raghav sorry that I'm getting to this late. Had a conference and retreat so i've been busy. We can't accept PR's with images. 
Can you:
- remove the images
- comment out path

The comment you've included in the yaml with a link to where they can download it, is more than enough!

Thanks again!

elvis!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 
 .DS_Store
+warpThemes.code-workspace
```
Comment (elviskahoro) [inline .gitignore:3]: Please remove

File: Cyberpunk/2077/yaml/Cyberpunk_2077_-_Night_City.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+background_image:
+  # background image source: https://cyberpunk.city/t/cdpr-the-new-engine-of-cyberpunk-2077-helps-develop-the-first-person-shooting-it-has-achieved-the-special-effect-of-ray-tracing/74
+  path: Cyberpunk/2077/Images/Night_City.jpg
```
Comment (elviskahoro) [inline Cyberpunk/2077/yaml/Cyberpunk_2077_-_Night_City.yaml:3]: Please comment out.

File: Cyberpunk/2077/yaml/Cyberpunk_2077_-_V.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+background_image:
+  # background image source: hhttps://www.insider.com/cyberpunk-2077-gameplay-demo-e3-details-release-date-2018-6
+  path: Cyberpunk/2077/Images/V.jpg
```
Comment (elviskahoro) [inline Cyberpunk/2077/yaml/Cyberpunk_2077_-_V.yaml:3]: Please comment out

File: Cyberpunk/Edgerunners/yaml/Edgerunners_-_David.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+background_image:
+  # background image source: https://wallpapersden.com/cyberpunk-netflix-2022-wallpaper/
+  path: Cyberpunk/Edgerunners/Images/David.jpg
```
Comment (elviskahoro) [inline Cyberpunk/Edgerunners/yaml/Edgerunners_-_David.yaml:3]: Please comment out!

File: Cyberpunk/Edgerunners/yaml/Edgerunners_-_Lucy_and_David_Jump.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+background_image:
+  # background image source: https://store.steampowered.com/news/group/38496645/view/3368143418771095467?l=schinese
+  path: Cyberpunk/Edgerunners/Images/Lucy_and_David_Jump.jpg
```
Comment (elviskahoro) [inline Cyberpunk/Edgerunners/yaml/Edgerunners_-_Lucy_and_David_Jump.yaml:3]: please comment out

File: Cyberpunk/README.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Screenshots
+
+|Theme name | Preview |
+| :---: | :---: |
+| [Cyberpunk 2077 - Night City](https://github.com/rexgraystone/warpCyberpunk/blob/Cyberpunk/Cyberpunk/2077/yaml/Cyberpunk_2077_-_Night_City.yaml) | ![Cyberpunk 2077 - Night City](Images/Cyberpunk2077_Night_City.png) |
+| [Cyberpunk 2077 - V](https://github.com/rexgraystone/warpCyberpunk/blob/Cyberpunk/Cyberpunk/2077/yaml/Cyberpunk_2077_-_V.yaml)  | ![Cyberpunk 2077 - V](Images/Cyberpunk2077_V.png) |
+| [Edgerunners - David](https://github.com/rexgraystone/warpCyberpunk/blob/Cyberpunk/Cyberpunk/Edgerunners/yaml/Edgerunners_-_David.yaml)  | ![Edgerunners - David](Images/Edgerunners_David.png) |
+| [Edgerunners - Lucy and David Jump](https://github.com/rexgraystone/warpCyberpunk/blob/Cyberpunk/Cyberpunk/Edgerunners/yaml/Edgerunners_-_Lucy_and_David_Jump.yaml) | ![Edgerunners - Lucy and David Jump](Images/Edgerunners_Lucy_and_David_Jump.png) |
```
Comment (elviskahoro) [inline Cyberpunk/README.md:8]: Please remove this as well.

**Review by elviskahoro — CHANGES_REQUESTED** [review]
Thanks in advance!

**Comment by elviskahoro** [comment]
@rexgraystone 
We need to comment out: "opacity" as well since this is for when we do have background images.

**Comment by rexgraystone** [comment]
> @rexgraystone We need to comment out: "opacity" as well since this is for when we do have background images.

I have updated. Please check for the same

**Comment by elviskahoro** [comment]
Let's ship it!

**Review by elviskahoro — APPROVED** [review]

**Pull request merged**